README.md

```diff
@@ -31,32 +31,33 @@ Each version of the plugin is published with a [specific set of versions of the
+| `flushMetricsToLogs`          | Send custom metrics by using logs with the Datadog Forwarder Lambda function (recommended). Defaults to `true`. If you disable this parameter, you must set `DATADOG_API_KEY` in your environment. To use an encrypted key, set `apiKMSKey` or `apiKeySecretArn` in the configuration. `flushMetricsToLogs` is ignored when `addExtension` is true.                                                                                 |
+| `site`                        | Set which Datadog site to send data to. This is only used when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com`, `us5.datadoghq.com`, and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                      |
+| `apiKey`                      | Datadog API key. Only needed when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Not required when the DATADOG_API_KEY environment variable is set. Defining `apiKey` adds the Datadog API key directly to your Lambda functions as an environment variable. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                |
+| `appKey`                      | Datadog app key. Only needed when `monitors` is defined. Not required when the DATADOG_APP_KEY environment variable is set.                                                                                                                                                                                                                                                                                                         |
+| `apiKeySecretArn`             | The ARN of the secret storing the Datadog API key in AWS Secrets Manager. Use this parameter if you're storing your secrets in AWS Secrets Manager, and when `flushMetricsToLogs` is `false` or `addExtension` is `true`. If set, remember to add the `secretsmanager:GetSecretValue` permission to the Lambda execution role.                                                                                                      |
+| `apiKMSKey`                   | Datadog API key encrypted using KMS. Use this parameter if you're storing your secrets with KMS, and if `flushMetricsToLogs` is `false` or `addExtension` is `true`. Defining `apiKMSKey` adds the Datadog API Key directly to your Lambda functions as an environment variable.                                                                                                                                                    |
```

Comment:
I think the different types of API key settings have gotten a bit confusing. It could be a good idea to clarify the situation a bit, maybe reorganize this part to try to make things more clear.